src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -427,6 +427,51 @@ def _rbd_mirror_rm(self, name=None):
         orchestrator.raise_if_exception(completion)
         return HandleCommandResult(stdout=completion.result_str())
 
+    @orchestrator._cli_write_command(
+        'orchestrator node_exporter add',
+        'name=num,type=CephInt,req=false '
+        'name=hosts,type=CephString,n=N,req=true '
+        'name=label,type=CephString,req=false',
+        'Create a node-exporter service'
+    )
+    def _node_exporter_add(self, count=None, hosts=None, label=None):
+        spec = orchestrator.StatefulServiceSpec(
+            placement=orchestrator.PlacementSpec(label=label, hosts=hosts, count=count))
+        completion = self.add_node_exporter(spec)
+        self._orchestrator_wait([completion])
+        orchestrator.raise_if_exception(completion)
+        return HandleCommandResult(stdout=completion.result_str())
+
+    @orchestrator._cli_write_command(
+        'orchestrator node_exporter update',
+        'name=num,type=CephInt,req=false '
+        'name=hosts,type=CephString,n=N,req=true '
+        'name=label,type=CephString,req=false',
+        'Update a node-exporter service'
+    )
+    def _node_exporter_update(self, count=None, hosts=None, label=None):
+        spec = orchestrator.StatefulServiceSpec(
+            placement=orchestrator.PlacementSpec(label=label, hosts=hosts, count=count))
+        completion = self.update_node_exporter(spec)
+        self._orchestrator_wait([completion])
+        orchestrator.raise_if_exception(completion)
+        return HandleCommandResult(stdout=completion.result_str())
+
+    @orchestrator._cli_write_command(
+        'orchestrator node_exporter rm',
+        'name=num,type=CephInt,req=false '
+        'name=hosts,type=CephString,n=N,req=true '
+        'name=label,type=CephString,req=false',
+        'Remove a node-exporter service'
+    )
+    def _node_exporter_remove(self, count=None, hosts=None, label=None):
```

Comment:
```
mypy run-test: commands[0] | mypy --config-file=../../mypy.ini ansible/module.py cephadm/module.py mgr_module.py mgr_util.py orchestrator.py orchestrator_cli/module.py progress/module.py rook/module.py test_orchestrator/module.py
orchestrator_cli/module.py: note: In member "_node_exporter_remove" of class "OrchestratorCli":
orchestrator_cli/module.py:470: error: Argument 1 to "remove_node_exporter" of "Orchestrator" has incompatible type "StatefulServiceSpec"; expected "str"
cephadm/module.py: note: In function "trivial_result":
cephadm/module.py:260: error: Invalid type comment or annotation
cephadm/module.py:260: note: Suggestion: use AsyncCompletion[...] instead of AsyncCompletion(...)
Found 2 errors in 2 files (checked 9 source files)
```